### PR TITLE
Portuguese strings v1

### PR DESCRIPTION
**Pull request by nmadruga**
PR for front-end https://github.com/openmsupply/application-manager-web-app/pull/979

### Changes:
Add local: pt-pt to generatePDF so dates show in the right format

Screen Shot 2021-09-10 at 6 18 16 PM

### CODE REVIEW

**Comment by CarlosNZ** [comment]
Looks cool :) 

Are you still having trouble with the Date formatting? (Is that why it's circled?)

**Comment by nmadruga** [comment]
> Looks cool :)
> 
> Are you still having trouble with the Date formatting? (Is that why it's circled?)

No, I sorted that out - just circled to dislay what was this about. I think this is good to merge since it's defined per action/template

File: plugins/action_generate_document/src/generateDoc.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ const generateDoc: ActionPluginType = async ({
     const result = await generatePDF({
       fileId: docTemplateId,
       data,
+      options: {
+        lang: 'pt-pt',
+      },
```
Comment (CarlosNZ) [inline plugins/action_generate_document/src/generateDoc.ts:23]: Doesn't this need to be a parameter, not hard-coded?

Comment (nmadruga) [reply plugins/action_generate_document/src/generateDoc.ts:23]: Yes - good point. Done in a hurry. I'll fix today

**Pull request closed without merging**